frontend/src/PrivateRoute.tsx

```diff
@@ -5,9 +5,14 @@ import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
 interface IPrivateRouteProps extends RouteProps {
   // tslint:disable-next-line:no-any
   component: any;
-  role?: string;
+  role?: string | Array<string>;
```

Comment:
I thought we had a single role per user? "single role, single agency" wasn't it?

Reply:
We enforce one role and one agency per user in the frontend.  However if a user belongs to a parent agency, they will have an array of agencies.

The code above is to allow a role to use it.  Which means more than one role could use the route.